.dockerignore

```diff
@@ -0,0 +1,9 @@
+codecov.yml
```

Comment:
Most of the things here are not large and don't really impact the time it takes to create the build context except for "examples" if we've run a chuck of those locally (e.g. my sub-directory is >600MB). Since we are also using the repo root as the build context, `.git` should also be ignored. The content of this file could likely just be
```
/.git/
/examples/
```